nmdc_runtime/api/db/mongo.py

```diff
@@ -107,3 +110,28 @@ def mongodump_excluded_collections():
         )
     )
     return excluded_collections
+
+
+def mongorestore_collection(mdb, collection_name, bson_file_path):
```

Comment:
I recommend adding a docstring to this function.

```suggestion
def mongorestore_collection(mdb, collection_name, bson_file_path):
    r"""
    Replaces the specified collection with one that reflects the contents of the 
    specified BSON file.
    """
```